tests/load/pipeline/test_import_export_schema.py

```diff
@@ -0,0 +1,109 @@
+import dlt, os
+
+from dlt.common.utils import uniq_id
+
+from tests.utils import TEST_STORAGE_ROOT
+from dlt.common.schema import Schema
+from dlt.common.storages.schema_storage import SchemaStorage
+
+
+IMPORT_SCHEMA_PATH = os.path.join(TEST_STORAGE_ROOT, "schemas", "import")
+EXPORT_SCHEMA_PATH = os.path.join(TEST_STORAGE_ROOT, "schemas", "export")
+
+
+EXAMPLE_DATA = [{"id": 1, "name": "dave"}]
+
+
+def _get_import_schema(schema_name: str) -> Schema:
+    return SchemaStorage.load_schema_file(IMPORT_SCHEMA_PATH, schema_name)
+
+
+def _get_export_schema(schema_name: str) -> Schema:
+    return SchemaStorage.load_schema_file(EXPORT_SCHEMA_PATH, schema_name)
+
+
+def test_schemas_files_get_created() -> None:
+    name = "schema_test" + uniq_id()
+
+    p = dlt.pipeline(
+        pipeline_name=name,
+        destination="duckdb",
+        import_schema_path=IMPORT_SCHEMA_PATH,
+        export_schema_path=EXPORT_SCHEMA_PATH,
+    )
+
+    p.run(EXAMPLE_DATA, table_name="person")
+
+    # basic check we have the table def in the export schema
+    export_schema = _get_export_schema(name)
+    export_schema.tables["person"]["columns"]["id"]["data_type"] == "bigint"
+    export_schema.tables["person"]["columns"]["name"]["data_type"] == "text"
+
+    # discovered columns are not present in the import schema
+    import_schema = _get_import_schema(name)
+    assert "id" not in import_schema.tables["person"]["columns"]
+    assert "name" not in import_schema.tables["person"]["columns"]
+
+
+def test_provided_columns_exported_to_import() -> None:
+    name = "schema_test" + uniq_id()
+
+    p = dlt.pipeline(
+        pipeline_name=name,
+        destination="duckdb",
+        import_schema_path=IMPORT_SCHEMA_PATH,
+        export_schema_path=EXPORT_SCHEMA_PATH,
+    )
+
+    p.run(EXAMPLE_DATA, table_name="person", columns={"id": {"data_type": "text"}})
+
+    # discovered columns are in export
```

Comment:
@rudolfix is this the correct behavior, that columns provided in the columns are not going to be in the exported schema?